pom.xml

```diff
@@ -820,6 +820,7 @@
           </dependency>
         </dependencies>
         <configuration>
+          <threshold>high</threshold>
```

Comment:
What about other threshold?

Reply:
It will report too many errors, and some of them you might not want to fix.

For example, the default threshold is `medium`, see "Summary of failures" in
https://github.com/apache/incubator-uniffle/runs/7299882996?check_suite_focus=true

Reply:
If there is a need, we can change it step by step, instead of trying to fix all of them in one huge PR.

Reply:
I think high is OK, LGTM.